From 9b95ff89abfde6071a16456f70eada468aaa2324 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 05 Dec 2018 13:52:18 +0200
Subject: [PATCH] preboot: shorten the namespace prefix

Switch from 'hl_preboot_' prefix to just 'preboot_'.
This makes the names shorter and simpler to read w/o any impact on
names collision or correctness.

Change-Id: Id264be36ab951f8c375ea3ce34464d9565682bd0
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index 22dcf11..e1f442e 100644
--- a/preboot.c
+++ b/preboot.c
@@ -56,7 +56,7 @@
 #define PREBOOT_VER_STR	"Preboot version " PREBOOT_VER \
 	" (" PREBOOT_DATE " - " PREBOOT_TIME ")"
 
-static void hl_preboot_puts_raw(const char* str)
+static void preboot_puts_raw(const char* str)
 {
 	int i;
 
@@ -66,13 +66,13 @@
 	}
 }
 
-static void hl_preboot_puts(const char* str)
+static void preboot_puts(const char* str)
 {
-	hl_preboot_puts_raw(str);
-	hl_preboot_puts_raw("\r");
+	preboot_puts_raw(str);
+	preboot_puts_raw("\r");
 }
 
-static void hl_preboot_puts_num(const char* log, unsigned int val, size_t size)
+static void preboot_puts_num(const char* log, unsigned int val, size_t size)
 {
 	int i;
 	char str[16], c;
@@ -92,20 +92,20 @@
 	str[i+2] = '\n';
 	str[i+3] = '\0';
 
-	hl_preboot_puts_raw(log);
-	hl_preboot_puts(str);
+	preboot_puts_raw(log);
+	preboot_puts(str);
 }
 
-static inline void hl_preboot_puts_char(const char* str, unsigned char val)
+static inline void preboot_puts_char(const char* str, unsigned char val)
 {
-	hl_preboot_puts_num(str, val, sizeof(val) * 2);
+	preboot_puts_num(str, val, sizeof(val) * 2);
 }
 
 /*
  * Using scratchpad registers for
  * passing flash layout to CPU SW
  */
-static void hl_preboot_copy_prepare(unsigned int *spl_off)
+static void preboot_copy_prepare(unsigned int *spl_off)
 {
 	unsigned int spl_offset = 0xa0000;
 	unsigned int uboot_offset = 0xb0000;
@@ -143,9 +143,9 @@
 		spl_image = "secondary";
 	}
 
-	hl_preboot_puts_raw("Preparing to run SPL ");
-	hl_preboot_puts_raw(spl_image);
-	hl_preboot_puts("\n");
+	preboot_puts_raw("Preparing to run SPL ");
+	preboot_puts_raw(spl_image);
+	preboot_puts("\n");
 
 	*spl_off = image_base_offset + spl_offset;
 	writel((image_base_offset + uboot_offset), mmPSOC_GLOBAL_CONF_SCRATCHPAD_30);
@@ -313,26 +313,26 @@
 	writel(0x00000001, mmPSOC_SPI_SSIENR);
 }
 
-static void hl_preboot_copy_spl(void)
+static void preboot_copy_spl(void)
 {
 	unsigned int  spl_offset;
 	unsigned char sr;
 
-	hl_preboot_copy_prepare(&spl_offset);
+	preboot_copy_prepare(&spl_offset);
 
 	hl_spi_init();
 	hl_spi_read_spl(spl_offset);
 	sr = hl_spi_prot();
-	hl_preboot_puts("SPL copying from flash is completed\n");
+	preboot_puts("SPL copying from flash is completed\n");
 	if ((sr & PROT_MASK) == PROT_MASK)
-		hl_preboot_puts("Flash may be protected, depend on WP# signal\n");
+		preboot_puts("Flash may be protected, depend on WP# signal\n");
 	else
-		hl_preboot_puts("Flash is not protected\n");
+		preboot_puts("Flash is not protected\n");
 }
 
 #ifdef DEV_IS_GOYA
 /* Handle BTL errate: exhaust timeout counter before any SPI access */
-static void hl_preboot_delay(void)
+static void preboot_delay(void)
 {
 	unsigned long cntpct, end;
 	int sysclk = SYS_REF_CLOCK, i;
@@ -352,7 +352,7 @@
 	}
 }
 #else
-#define hl_preboot_delay()
+#define preboot_delay()
 #endif
 
 #if defined (DEV_IS_GOYA) && !defined(TARGET_IS_PLDM)
@@ -415,10 +415,10 @@
 #endif
 
 /* Handle PCIe errata: make sure reset_n de-asserted only after perst */
-static void hl_preboot_pcie_init(void)
+static void preboot_pcie_init(void)
 {
 	if (readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS) & SKIP_PCIE_INIT) {
-		hl_preboot_puts("PCIe initialization is skipped!\n");
+		preboot_puts("PCIe initialization is skipped!\n");
 		return;
 	}
 
@@ -465,37 +465,37 @@
 		}
 	}
 
-	hl_preboot_puts("PCIe link is UP!\n");
+	preboot_puts("PCIe link is UP!\n");
 }
 
 static const char *preboot_ver_str = PREBOOT_VER_STR;
 
 #ifndef TARGET_IS_TESTER
-static void hl_preboot_version(void)
+static void preboot_version(void)
 {
 	u64 offset = (u64) preboot_ver_str;
 
-	hl_preboot_puts("\n");
-	hl_preboot_puts_raw(preboot_ver_str);
-	hl_preboot_puts("\n");
+	preboot_puts("\n");
+	preboot_puts_raw(preboot_ver_str);
+	preboot_puts("\n");
 
 	writel((u32)(offset & ~CONFIG_SYS_SDRAM_BASE),
 			mmPSOC_GLOBAL_CONF_SCRATCHPAD_28);
 }
 #else	/* TARGET_IS_TESTER */
-static inline void hl_preboot_version(void) {}
+static inline void preboot_version(void) {}
 #endif	/* TARGET_IS_TESTER */
 
-void hl_preboot_main(void)
+void preboot_main(void)
 {
 	void (*spl_start)(void) = (void*)CONFIG_SPL_TEXT_BASE;
 
 	writel(CPU_BOOT_STATUS_IN_PREBOOT, mmPSOC_GLOBAL_CONF_WARM_REBOOT);
 
-	hl_preboot_version();
-	hl_preboot_pcie_init();
-	hl_preboot_delay();
-	hl_preboot_copy_spl();
+	preboot_version();
+	preboot_pcie_init();
+	preboot_delay();
+	preboot_copy_spl();
 
 	spl_start();
 }
diff --git a/start.S b/start.S
index 1757001..44d31f0 100644
--- a/start.S
+++ b/start.S
@@ -15,4 +15,4 @@
 	ldr	x0, =(_start + 0x10000)
 	bic	sp, x0, #0xf		/* 16-byte alignment for ABI compliance */
 	mov	x29, sp			/* Set FP (x29) */
-	bl	hl_preboot_main		/* Set LR (x30) */
+	bl	preboot_main		/* Set LR (x30) */
